woocommerce/woocommerce-android
# 4952 cardreader battery level seems to be out dated

Closes: #4952


### Description

The PR utilizes a feature of the SDK v2.7.0 that adds callback that invokes when a reader's batter status changes 

### Testing instructions

* Collect to the reader via settings
* Wait 10 minutes (yes, the SDK claims that it updates it once per 10 minutes)
* Notice that battery level dropped
